src/effects/native/reverbeffect.cpp

```diff
@@ -74,48 +99,15 @@ void ReverbEffect::processChannel(const ChannelHandle& handle,
     Q_UNUSED(handle);
     Q_UNUSED(enableState);
     Q_UNUSED(groupFeatures);
-    Q_UNUSED(sampleRate);
-    CSAMPLE bandwidth = m_pBandWidthParameter->value();
-    CSAMPLE damping = m_pDampingParameter->value();
 
-    // Flip value around.  Assumes max allowable is 1.0.
-    damping = 1.0 - damping;
+    const auto bandwidth = m_pBandWidthParameter->value();
+    const auto decay = m_pDecayParameter->value();
+    const auto damping = m_pDampingParameter->value();
+    const auto blend = m_pBlendParameter->value();
 
-    bool params_changed = (damping != pState->prev_damping ||
-                           bandwidth != pState->prev_bandwidth);
+    // Set the sample rate (TODO XXX: Maybe we want to call .init() after this?)
```

Comment:
We do need to call ```init()``` otherwise it has no effect (looks like fs is only used in ```init()```).

Maybe you could call it in setSampleRate if fs != sampleRate, or by adding a getSampleRate method and calling init from here if it doesn't match sampleRate.